src/Map.jsx

```diff
@@ -57,9 +58,10 @@ function Map() {
             {generateLocations(50, { center: geolocation, radius: 5 }).map(
               (location, index) => (
                 <GrommetMarker key={index} position={location} />
-              ),
+              )
             )}
           </MarkerClusterGroup>
+          <GeoJSON key="my-geojson" data={geojsonLocations} />
```

Comment:
quick question -- is it expected that these will be clusterable?

Reply:
Yes, we should be able to cluster the geojson point locations, haven't tested that yet though

Reply:
Cool -- would it be worth trying in this PR? or are you thinking separately?

Reply:
Yeah I can try it out in this PR

Reply:
You should be able to wrap in `<MarkerClusterGroup>` either in its own cluster group, or add it as a child of the one preceding.

Reply:
Just pushed up a commit that clusters the geojson markers